src/Raven.Server/Documents/Replication/Incoming/AbstractIncomingReplicationHandler.cs

```diff
@@ -85,6 +88,7 @@ public abstract class AbstractIncomingReplicationHandler<TContextPool, TOperatio
             ConnectionInfo = IncomingConnectionInfo.FromGetLatestEtag(replicatedLastEtag);
             SupportedFeatures = TcpConnectionHeaderMessage.GetSupportedFeaturesFor(tcpConnectionOptions.Operation, tcpConnectionOptions.ProtocolVersion);
             ConnectionInfo.RemoteIp = ((IPEndPoint)_tcpClient.Client.RemoteEndPoint)?.Address.ToString();
+            ReplicationType = replicationType;
```

Comment:
do we need that only for `GetReplicationPerformanceType` ?

Reply:
Yes.. any idea how to avoid it? :-)

Reply:
done